Failing build - Unable to extract LWJGL natives

@NathanSweet Please see the comments on commit https://github.com/libgdx/libgdx/commit/e1aa0e56415f76df6de67aaafc4ce3bc2cc4f9c1. Several people have reported the same error, now on Windows, Mac, & Linux, from this change. Here's my stack trace on OS X Yosemite:

```
com.badlogic.gdx.utils.GdxRuntimeException: Unable to extract LWJGL natives.
	at com.badlogic.gdx.backends.lwjgl.LwjglNativesLoader.load(LwjglNativesLoader.java:67) ~[gdx-backend-lwjgl-1.7.2-SNAPSHOT.jar:na]
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.<init>(LwjglApplication.java:76) ~[gdx-backend-lwjgl-1.7.2-SNAPSHOT.jar:na]
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.<init>(LwjglApplication.java:64) ~[gdx-backend-lwjgl-1.7.2-SNAPSHOT.jar:na]
	at com.forerunnergames.peril.desktop.DesktopLauncher.main(DesktopLauncher.java:65) ~[main/:na]
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Error extracting file: openal.dylib
To: /var/folders/9g/w7dzqgq94lj2q7kd_n7q7rth0000gn/T/libgdxAaron/f7649394
	at com.badlogic.gdx.utils.SharedLibraryLoader.extractFile(SharedLibraryLoader.java:265) ~[gdx-1.7.2-SNAPSHOT.jar:na]
	at com.badlogic.gdx.utils.SharedLibraryLoader.extractFileTo(SharedLibraryLoader.java:171) ~[gdx-1.7.2-SNAPSHOT.jar:na]
	at com.badlogic.gdx.backends.lwjgl.LwjglNativesLoader.load(LwjglNativesLoader.java:60) ~[gdx-backend-lwjgl-1.7.2-SNAPSHOT.jar:na]
	... 3 common frames omitted
Caused by: java.io.FileNotFoundException: /var/folders/9g/w7dzqgq94lj2q7kd_n7q7rth0000gn/T/libgdxAaron/f7649394 (Is a directory)
	at java.io.FileOutputStream.open(Native Method) ~[na:1.7.0_65]
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221) ~[na:1.7.0_65]
	at java.io.FileOutputStream.<init>(FileOutputStream.java:171) ~[na:1.7.0_65]
	at com.badlogic.gdx.utils.SharedLibraryLoader.extractFile(SharedLibraryLoader.java:255) ~[gdx-1.7.2-SNAPSHOT.jar:na]
```

--- a/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglNativesLoader.java
+++ b/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglNativesLoader.java
@@ -53,15 +53,22 @@ public final class LwjglNativesLoader {
 		try {
 			if (isWindows) {
 				nativesDir = loader.extractFile(is64Bit ? "lwjgl64.dll" : "lwjgl.dll", null).getParentFile();
-				if (!LwjglApplicationConfiguration.disableAudio)
-					loader.extractFileTo(is64Bit ? "OpenAL64.dll" : "OpenAL32.dll", nativesDir);
+				if (!LwjglApplicationConfiguration.disableAudio) {
+					String windowsOpenALFile = is64Bit ? "OpenAL64.dll" : "OpenAL32.dll";
+					loader.extractFileTo(windowsOpenALFile, new File(nativesDir, windowsOpenALFile));
+				}
 			} else if (isMac) {
 				nativesDir = loader.extractFile("liblwjgl.dylib", null).getParentFile();
-				if (!LwjglApplicationConfiguration.disableAudio) loader.extractFileTo("openal.dylib", nativesDir);
+				if (!LwjglApplicationConfiguration.disableAudio) {
+					String macOpenALFile = "openal.dylib";
+					loader.extractFileTo(macOpenALFile, new File(nativesDir, macOpenALFile));
+				}
 			} else if (isLinux) {
 				nativesDir = loader.extractFile(is64Bit ? "liblwjgl64.so" : "liblwjgl.so", null).getParentFile();
-				if (!LwjglApplicationConfiguration.disableAudio)
-					loader.extractFileTo(is64Bit ? "libopenal64.so" : "libopenal.so", nativesDir);
+				if (!LwjglApplicationConfiguration.disableAudio) {
+					String linuxOpenALFile = is64Bit ? "libopenal64.so" : "libopenal.so";
+					loader.extractFileTo(linuxOpenALFile, new File(nativesDir, linuxOpenALFile));
+				}
 			}
 		} catch (Throwable ex) {
 			throw new GdxRuntimeException("Unable to extract LWJGL natives.", ex);

